apps/wallet/src/ui/app/hooks/useTransactionData.ts

```diff
@@ -29,7 +28,7 @@ export function useTransactionData(sender?: string | null, transaction?: Transac
 export function useTransactionGasBudget(sender?: string | null, transaction?: Transaction | null) {
     const { data, ...rest } = useTransactionData(sender, transaction);
 
-    const [formattedGas] = useFormatCoin(data?.gasData.budget, IOTA_TYPE_ARG, CoinFormat.FULL);
+    const [formattedGas] = useFormatCoin(data?.gasData.budget, CoinFormat.FULL);
```

Comment:
Here I think it should not be removed, because the second param corresponds to `coinType`, and the third to coin format

Reply:
Good point, maybe we could make the optional params of useFormatCoin an object instead of plain arguments.